src/wallet/rpcwallet.cpp

```diff
@@ -732,7 +732,7 @@ static UniValue getbalance(const JSONRPCRequest& request)
                 {
                     {"dummy", RPCArg::Type::STR, RPCArg::Optional::OMITTED_NAMED_ARG, "Remains for backward compatibility. Must be excluded or set to \"*\"."},
                     {"minconf", RPCArg::Type::NUM, /* default */ "0", "Only include transactions confirmed at least this many times."},
-                    {"include_watchonly", RPCArg::Type::BOOL, /* default */ "false", "Also include balance in watch-only addresses (see 'importaddress')"},
+                    {"include_watchonly", RPCArg::Type::BOOL, /* default */ "false, true for watchonly wallets", "Also include balance in watch-only addresses (see 'importaddress')"},
```

Comment:
Suggestion for the RPCHelpMan default messages: `"true for watch-only wallets, otherwise false"`

Grepping related cases in the codebase comes up with similar, e.g. src/wallet/rpcdump.cpp:120

As for "watchonly" vs "watch-only", in the descriptions on the same line it is dasherized. Unless other reviewers have a different opinion, it might be more clear to do the same?

Reply:
yeah that message sounds better. I'll go with `watch-only` as well.

Reply:
done